app/views/internal/articles/_individual_article.html.erb

```diff
@@ -127,17 +127,15 @@
         <div class="row" style="font-size:15px;">
           <%= article.processed_html&.html_safe %>
         </div>
-        <div class="inner-row" style="padding:10px;margin-top:20px;font-weight:800;">
+        <div class="flex-column" style="padding:10px;margin-top:20px;font-weight:800;">
           <div>
             <%= form_tag "/internal/buffer_updates" do %>
               <input type="hidden" name="social_channel" value="main_twitter" />
               <input type="hidden" name="article_id" value="<%= article.id %>" />
               <p> Twitter MAIN</p>
               <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255"><%= article.title %>
-                <% if !article.user.twitter_username.blank? %>
-
-{ author: @<%= article.user.twitter_username %> } #DEVCommunity
-                <% end %></textarea>
+<% if !article.user.twitter_username.blank? %>&#013{ author: @<%= article.user.twitter_username %> } #DEVCommunity<% end %>
```

Comment:
Could you fix the indentation here? Btw the check could be refactored to `if article.user.twitter_username?`

Reply:
@lightalloy Thanks for the review! Fixed the condition.
I'd love to fix the indentation, but I haven't fount any instrument to do so and don't have extra characters inside the `<textarea>`. I usually use `HEREDOC`, but it can't be used here with variables.
Do you have any ideas on how we can remove the extra characters and fix the indentation?
Thanks in advance!

Reply:
I see, I tried indenting the line, but haven't noticed the extra characters, sorry. Let's keep it as is.
I don't see any changes in the condition, have you pushed them?

Reply:
```suggestion
                <% if !article.user.twitter_username.blank? %>&#013{ author: @<%= article.user.twitter_username %> } #DEVCommunity<% end %>
```

Reply:
@rhymes that adds blank characters to the `textarea` text.

Reply:
😅

Reply:
@lightalloy I definetely forget to commit the changes and I found another space which I forget to commit :) Made the changes in follow up PR if it's okay https://github.com/thepracticaldev/dev.to/pull/4437